tools/nxstyle.c

```diff
@@ -135,67 +211,98 @@ int main(int argc, char **argv, char **envp)
   int externc_lineno;   /* Last line where 'extern "C"' declared */
   int linelen;          /* Length of the line */
   int maxline;          /* Lines longer that this generate warnings */
+  bool silent;          /* Used with go not go test option */
   int n;
   int i;
+  int c;
+  extern char *optarg;
+  extern int optopt;
+  extern int optind;
 
```

Comment:
opt* Already declared in unistd.h, why need again?

Reply:
You have good eyes.  Yes, those externs must go.

Reply:
Done!